src/styles/modal.css

```diff
@@ -0,0 +1,108 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: var(--box-shadow-5);
+}
+
+.ui.modal > .icon:first-child + * {
+  border-radius: 0;
+}
+
+/* close icon */
+
+.ui.modal > .close {
+  color: var(--shade-white);
+  font-size: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  right: var(--spacing-2);
+  top: var(--spacing-3);
+  transition: opacity var(--transition-hover);
+  width: 1.5rem;
+}
+
+@media only screen and (max-width: 991px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
```

Comment:
Do we need to repeat these styles for each break point?

Reply:
Yes, otherwise it doesn't pick them up. Same was for the input, when we had to do it for each input type.